operator/go.mod

```diff
@@ -0,0 +1,14 @@
+module sigs.k8s.io/cluster-api/operator
```

Comment:
I would reccomand to drop this go mod file so we will get automatic dependency version alignment with the rest of the codebase

Reply:
Done. Dropped in the next PR.